cluster/manifests/ingress-controller/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         operator: Exists
       containers:
       - name: controller
-        image: registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:v0.3.5
+        image: registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:v0.3.5
```

Comment:
I also think the former sounds better, but the repository is called `kube-ingress-aws-controller` so it makes sense to keep it consistent like @linki did :+1: